File: ulohy/04-velikost-pismene/reseni/main.py
import sys

if __name__ == "__main__":
    line = input()
    letter = line[0]
    charCode = ord(letter)

    output = ""

    if 65 <= charCode <= 90:
        output = "upper"
    elif 97 <= charCode <= 122:
        output = "lower"

    print(output)

File: ulohy/04-velikost-pismene/reseni/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on("line", (line) => {
    let charCode = line.charCodeAt(0);

    let output = "";

    if (charCode >= 65 && charCode <= 90) {
        output = "upper";
    } else if (charCode >= 97 && charCode <= 122) {
        output = "lower";
    }

    console.log(output);

    rl.close();
});
